src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/base/GT_MetaTileEntity_MultiblockBase_EM.java

```diff
@@ -118,6 +118,12 @@ public abstract class GT_MetaTileEntity_MultiblockBase_EM extends GT_MetaTileEnt
     // data required to operate
     protected long eRequiredData = 0;
 
+    // Counter for the computation timeout. Will be initialized one to the max time and then only decreased.
+    protected int eComputationTimeout = MAX_COMPUTATION_TIMEOUT;
+
+    // Max timeout of computation in ticks
+    protected static int MAX_COMPUTATION_TIMEOUT = 40;
```

Comment:
Pretty sure the default value for mStartUpCheck has always been 100. Are you sure a timeout of 40 is enough?

Reply:
The timeout is only triggered after all structure checks have been done. It is only to compensate for when the research station happens to be loaded before the quantum computer. But I can increase the time some more, sure.

Reply:
If that is the case then we are probably good to merge. 2 seconds should be enough for most setups.